src/Microsoft.HttpRepl/Commands/HelpCommand.cs

```diff
@@ -186,6 +186,7 @@ public static void CoreGetHelp(IShellState shellState, ICommandDispatcher<HttpSt
             output.AppendLine();
             output.AppendLine($"{"connect",navCommandColumn}{GetCommand<ConnectCommand>(dispatcher).GetHelpSummary(shellState, programState)}");
             output.AppendLine($"{"set header",navCommandColumn}{GetCommand<SetHeaderCommand>(dispatcher).GetHelpSummary(shellState, programState)}");
+            output.AppendLine($"{"set queryString",navCommandColumn}{GetCommand<SetQueryStringCommand>(dispatcher).GetHelpSummary(shellState, programState)}");
```

Comment:
From the output of this, it looks like this is now our longest command name, so we'll need to bump `navCommandColumn` (probably bump it by 2 since the command will be longer when it is changed to `query-string`):

```console
Setup Commands:
Use these commands to configure the tool for your API server

connect        Configures the directory structure and base address of the api server
set header     Sets or clears a header for all requests. e.g. `set header content-type application/json`
set queryStringSets or clears a query string key and value. When [value] is empty the header is cleared. The value will be UrlEncoded.
```